ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -17,15 +17,12 @@ import {
   idPayInitiativesFromInstrumentRefreshStop
 } from "../../../idpay/wallet/store/actions";
 import { idPayEnabledInitiativesFromInstrumentSelector } from "../../../idpay/wallet/store/reducers";
-import { IOStyles } from "../../../../components/core/variables/IOStyles";
-import SectionHeader from "../../../../components/services/SectionHeader";
+import { VSpacer } from "../../../../components/core/spacer/Spacer";
```

Comment:
Like above

Reply:
adressed both in [e4b8d7b](https://github.com/pagopa/io-app/pull/4867/commits/e4b8d7b5ef8ca52ffa037af296ea457f8d6f6b8a)